test/forms/labels/input-label.test.js

```diff
@@ -1,93 +1,109 @@
 import React from 'react'
-import { mount } from 'enzyme'
 import { InputLabel } from '../../../src/'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 
 const name = 'person.firstName'
 const formattedName = 'First Name'
 
 test('when label is false - does not render a label', () => {
-  const wrapper = mount(<InputLabel name={name} label={false} />)
-  expect(wrapper.find('label').exists()).toEqual(false)
+  render(<InputLabel name={name} label={false} />)
+  expect(screen.queryByText(formattedName)).not.toBeInTheDocument()
 })
 
 test('when label not provided - renders a label with content equal to formatted input name', () => {
-  const wrapper = mount(<InputLabel name={name} />)
-  expect(wrapper.find('label').text()).toEqual(formattedName)
+  render(<InputLabel name={name} />)
+  expect(screen.getByText(formattedName)).toBeInTheDocument()
 })
 
 test('when label provided - renders a label with content equal to string', () => {
-  const wrapper = mount(<InputLabel name={name} label="foo" />)
-  expect(wrapper.find('label').text()).toEqual('foo')
+  render(<InputLabel name={name} label="foo" />)
+  expect(screen.getByText('foo')).toBeInTheDocument()
 })
 
 test('when children are provided, renders a label with content equal to children', () => {
-  const onClick = jest.fn()
-  const wrapper = mount(
+  render(
     <InputLabel name={name}>
-      Are you <span onClick={onClick}>sure</span>?
+      Are you sure?
     </InputLabel>
   )
-  expect(wrapper.find('label').text()).toEqual('Are you sure?')
+
+  expect(screen.getByText('Are you sure?')).toBeInTheDocument()
 })
 
-test('when children are provided, renders a label with custom interactions intact', () => {
+test('when children are provided, renders a label with custom interactions intact', async () => {
+  const user = userEvent.setup()
   const onClick = jest.fn()
-  const wrapper = mount(
+  render(
     <InputLabel name={name}>
       Are you{' '}
-      <span id="click" onClick={onClick}>
+      <span data-testid="click" onClick={onClick}>
         sure
       </span>
       ?
     </InputLabel>
   )
-  wrapper.find('#click').simulate('click')
+
+  await user.click(screen.getByTestId('click'))
+
   expect(onClick).toHaveBeenCalled()
 })
 
 test('when hint provided - shows hint', () => {
-  const wrapper = mount(<InputLabel name={name} hint="hint" />)
-  expect(wrapper.find('label > i').text()).toEqual('hint')
+  render(<InputLabel name={name} hint="hint" />)
+  expect(screen.getByText(formattedName)).toHaveTextContent('hint')
 })
 
 test('when tooltip provided - shows tooltip trigger', () => {
-  const wrapper = mount(<InputLabel name={name} tooltip="tooltip" />)
-  expect(wrapper.find('span.tooltip-trigger').exists()).toEqual(true)
+  render(<InputLabel name={name} tooltip="tooltip" />)
+  expect(screen.getByText(formattedName).nextSibling).toHaveClass('tooltip-trigger')
```

Comment:
This isn't very "RTLy", but I'm ok with this breaking if the tooltip markup gets changed. It's not an accessible implementation anyways, so I could see us ripping out entirely in the future for something like [Tippy.js](https://atomiks.github.io/tippyjs/)